packages/api/core/src/api/package.ts

```diff
@@ -206,9 +206,11 @@ export const listrPackage = ({
             },
             async (buildPath, electronVersion, pPlatform, pArch, done) => {
               const bins = await glob(path.join(buildPath, '**/.bin/**/*'));
-              for (const bin of bins) {
-                await fs.remove(bin);
-              }
+              await Promise.all(
+                bins.map(async (bin) => {
+                  await fs.remove(bin);
+                })
+              );
```

Comment:
```suggestion
              await Promise.all(
                bins.map(fs.remove)
              );
```

Reply:
See above. `fs.remove` takes a second parameter, this is unsafe

Reply:
for clarity- we should not merge this change request into the PR?